File: Examples/CSharp/NoteBook/ConvertToImage.cs
using System.IO;
using Aspose.Note;
using System;
using System.Collections.Generic;

namespace Aspose.Note.Examples.CSharp.WorkingWithNoteBook
{
    public class ConvertToImage
    {
        public static void Run()
        {
            // ExStart:ConvertToImage
            // ExFor:Notebook
            // ExSummary:Shows how to save notebook as image.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_NoteBook();

            // Load a OneNote Notebook
            var notebook = new Notebook(dataDir + "Notizbuch öffnen.onetoc2");

            dataDir = dataDir + "ConvertToImage_out.png";

            // Save the Notebook
            notebook.Save(dataDir);

            // ExEnd:ConvertToImage

            Console.WriteLine("\nNoteBook document converted to image successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/VisualBasic/NoteBook/ConvertToImage.vb
Imports System.IO
Imports Aspose.Note
Imports System.Collections.Generic

Namespace WorkingWithNoteBook
    Public Class ConvertToImage
        Public Shared Sub Run()
            ' ExStart:ConvertToImage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_NoteBook()

            ' Load a OneNote Notebook
            Dim notebook = New Notebook(dataDir & Convert.ToString("Notizbuch öffnen.onetoc2"))

            dataDir = dataDir & Convert.ToString("ConvertToImage_out_.png")

            ' Save the Notebook
            notebook.Save(dataDir)

            ' ExEnd:ConvertToImage
            Console.WriteLine(Convert.ToString(vbLf & "NoteBook document converted to image successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace